src/igroupadmin.cpp

```diff
@@ -333,8 +334,8 @@ main( int argc, char **argv ) {
 
     int argOffset;
 
-    int maxCmdTokens = 20;
-    char *cmdToken[20];
+    const int maxCmdTokens = 20;
+    char *cmdToken[maxCmdTokens];
```

Comment:
Do we need to clear the array?

Reply:
That happens a few lines down, each element gets an empty string by default. https://github.com/d-w-moore/irods_client_icommands/blob/1815fab27082a02d5cb36c1123953ffc11bf8c11/src/igroupadmin.cpp#L364

Reply:
Good enough for me.